src/globus_sdk/tokenstorage/v2/base.py

```diff
@@ -129,9 +117,16 @@ def _extract_identity_id(self, token_response: OAuthTokenResponse) -> str | None
 
 class FileTokenStorage(TokenStorage, metaclass=abc.ABCMeta):
     """
-    File adapters are for single-user cases, where we can assume that there's a
-    simple file-per-user and users are only ever attempting to read their own
-    files.
+    A base class for token storages which store tokens on a local file.
+
+    Common functionality for file-based token storages like file creation and class
+    instantiation for a GlobusApp is defined here.
```

Comment:
Maybe? There's another reference below this note, too.

```suggestion
    instantiation for a :class:`GlobusApp` is defined here.
```

Reply:
This won't resolve properly.
GlobusApp doesn't have reference documentation yet (that's next in my change list).